//

const getUsers = async (req: Request, res: Response) => {
    try {
        
    } catch (error) {
        // res.status(500).json({ message: 'Internal Error', error})
    }
}
const getUserById = (id: string) => {}
const createUser = () => {}
const updateUser = (id: string) => {}
const deleteUser = (id: string) => {}

export  {
    getUsers,
    createUser,
    getUserById,
    updateUser,
    deleteUser
}